### PR TITLE
update oc-mirror payload image

### PR DESCRIPTION
**Pull request by zhouying7780**

### CODE REVIEW

**Comment by zhouying7780** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-aws-ipi-disc-priv-f28-ota

File: ci-operator/step-registry/mirror-images/by-oc-mirror/mirror-images-by-oc-mirror-commands.sh [hunk 1]
```diff
@@ -98,53 +90,49 @@ extract_oc_mirror "${new_pull_secret}" "${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRI
...
+if [ ! -s "${idms_file}" ]; then
+    echo "${idms_file} not found, exit..."
     exit 1
+else
+    run_command "cat '$idms_file'"
+fi
+
+if [ -s "${itms_file}" ]; then
+    echo "${itms_file} found"
+    run_command "cat '$idms_file'"
```
Comment (jianlinliu) [inline ci-operator/step-registry/mirror-images/by-oc-mirror/mirror-images-by-oc-mirror-commands.sh:127]:
```suggestion
    run_command "cat '$itms_file'"
```

File: ci-operator/step-registry/mirror-images/by-oc-mirror/mirror-images-by-oc-mirror-commands.sh [hunk 2]
```diff
@@ -98,53 +90,49 @@ extract_oc_mirror "${new_pull_secret}" "${OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRI
...
 fi
+
 key_name="imageContentSources"
 if [[ "${ENABLE_IDMS}" == "yes" ]]; then
     key_name="imageDigestSources"
 fi
-yq-v4 --prettyPrint eval-all "{\"$key_name\": .spec.repositoryDigestMirrors}" "${icsp_file}" > "${install_config_mirror_patch}" || exit 1
-run_command "cp ${mapping_file} ${ARTIFACT_DIR}/mapping.txt"
+
+yq-v4 --prettyPrint eval-all "{\"$key_name\": .spec.imageDigestMirrors}" "${idms_file}" > "${install_config_mirror_patch}" || exit 1
```
Comment (jianlinliu) [inline ci-operator/step-registry/mirror-images/by-oc-mirror/mirror-images-by-oc-mirror-commands.sh:135]: To take itms file into consideration, need to move the line 125 - 127 here.
```
if [ -s "${itms_file}" ]; then
    echo "${itms_file} found"
    run_command "cat '$itms_file'"

    new_data=$(yq-v4 eval-all '.spec.imageTagMirrors' "${itms_file}") yq-v4 eval-all  ".$key_name += env(new_data)" "${install_config_mirror_patch}" || exit 1
fi
```

**Comment by zhouying7780** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.17-amd64-nightly-aws-ipi-disc-priv-f28-ota

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml [hunk 3]
```diff
@@ -125,7 +125,7 @@ tests:
       TEST_SCENARIOS: OTA
     test:
     - chain: openshift-e2e-test-qe-longduration
-    workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private
+    workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-sno-by-oc-mirror
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml:128]: pj-rehearse testing get passed, I think we can revert this change on this line, later I will try to add some dedicated  ci jobs to cover oc-mirror.

Comment (zhouying7780) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml]: @jianlinliu Done , thanks

**Pull request merged**